app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/repository/SessionRemoteRepository.java

```diff
@@ -1,5 +1,7 @@
 package br.org.cesar.discordtime.stickysessions.data.remote.repository;
 
+import android.util.Log;
```

Comment:
I don't think you are using this import. Could you please remove it?

Reply:
done